src/kalman/kalman.cpp

```diff
@@ -1,54 +1,60 @@
-#include <iostream>
-#include<eigen3/Eigen/Dense>
-#include <kalman.hpp>
-namespace huro{
-	KalmanFilter::KalmanFilter(
-			const Eigen::MatrixXd& A,
-			const Eigen::MatrixXd& B,
-			const Eigen::MatrixXd& C,
-			const Eigen::MatrixXd& Q,
-			const Eigen::MatrixXd& R,
-			const Eigen::MatrixXd& P)
-		: A(A), B(B), C(C), Q(Q), R(R), P0(P),
-			m(C.rows()), n(A.rows()), c(B.cols()), initialized(false),
-			I(n, n), x_hat(n)
-	{
-		I.setIdentity();
-	}
+// #include <iostream>
+// #include <iostream>
 
-	void KalmanFilter::init(const Eigen::VectorXd& x0) {
-		x_hat = x0;
-		P = P0;
-		initialized = true;
-	}
+#include "kalman.hpp"
```

Comment:
This might be the reason for the errors. CMakeLists will try to find kalman.hpp in the current directory if you do this.